src/System.Windows.Forms/src/System/Windows/Forms/KeysConverter.cs

```diff
@@ -30,7 +30,7 @@ private void Initialize()
 
     private void AddLocalizedKeyNames(CultureInfo cultureInfo)
     {
-        if (CultureToDisplayOrder.ContainsKey(cultureInfo) && CultureToDisplayOrder.ContainsKey(cultureInfo))
+        if (CultureToDisplayOrder.ContainsKey(cultureInfo))
```

Comment:
Thanks for bringing up attention to this, but I believe it was intended to do the following:
```suggestion
        if (CultureToDisplayOrder.ContainsKey(cultureInfo) && CultureToKeyName.ContainsKey(cultureInfo))
```

This method inserts values to both of these dictionaries, so it makes sense to check them both.

Reply:
That would make more sense indeed